### PR TITLE
[FEATURE] Updates API call for stores list

### PR DESCRIPTION
**Pull request by astr0-4**
* Adding tests for getStoreLocations Epic
 * renames and moves a couple files for organization

### CODE REVIEW

**Review by jocelynjeffrey — APPROVED** [review]

**Comment by jocelynjeffrey** [comment]
Looks good @astr0-4 !

File: src/tests/epics/getStoreLocations.epic.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { ActionsObservable } from "redux-observable"
+import "rxjs"
+import { Observable } from "rxjs/Observable"
+import "rxjs/add/observable/of"
+
+import { getStoreLocations } from "../../redux/epics/getStoreLocations.epic"
+import {
+  getStoresByProductIDs,
+  sendStoresByProductIDs,
```
Comment (jprittie) [inline src/tests/epics/getStoreLocations.epic.test.js]: Hello... this throws a warning, i.e. sendStoresByProductIDs is defined but never used

**Review by jprittie — APPROVED** [review]
I found one (small) warning in the epic test but otherwise it looks great.

File: src/tests/epics/getStoreLocations.epic.test.js (hunk 1 above)
Comment (astr0-4) [reply src/tests/epics/getStoreLocations.epic.test.js]: oh right! I saw that and then forgot to remove it. Good catch @jprittie!

**Pull request merged**